diff --git a/host/common/include/conversions.h b/host/common/include/conversions.h
index 39eee5d..1562cf7 100644
--- a/host/common/include/conversions.h
+++ b/host/common/include/conversions.h
@@ -33,6 +33,10 @@
 #include <libbladeRF.h>
 #include "host_config.h"
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+
 /**
  * Represents an association between a string suffix for a numeric value and
  * its multiplier. For example, "k" might correspond to 1000.
@@ -323,4 +327,8 @@ void float_to_sc16q11(const float *in, int16_t *out, unsigned int n);
  */
 bladerf_cal_module str_to_bladerf_cal_module(const char *str);
 
+#ifdef __cplusplus
+}
+#endif
+
 #endif
